service/src/main/java/uk/gov/mca/beacons/api/export/mappers/ExportMapper.java

```diff
@@ -298,6 +299,50 @@ public BeaconExportDTO toLegacyBeaconExportDTO(LegacyBeacon beacon) {
       .build();
   }
 
+  public BeaconExportDTO toLegacyBeaconExportDTO(BackupLegacyBeacon beacon) {
```

Comment:
I'm a little confused by this, what are the differences here?

Reply:
toLegacyBeaconExportDTO() takes a LegacyBeacon; when we're doing the backup, we're getting BeaconBackupItems . BackupLegacyBeacon contains just the fields relevant to a legacy beacon. We want to map that data into a format appropriate for a spreadsheet row, so that's why I'm using the mapping method. do you want me instead to map a BeaconBackupItem to a LegacyBeacon to enable us to just have the one version of toLegacyBeaconExportDTO() ?